src/golang/cmd/server/server/vault.go

```diff
@@ -44,7 +44,7 @@ func syncVaultWithStorage(
 		return err
 	}
 
-	if err := utils.MigrateVault(
+	if _, err := utils.MigrateVault(
```

Comment:
This call can ignore the list of keys returned, because a few lines down we delete the entire outdated vault directory anyways.